### ISSUE TITLE
Default behavior for parameter in post method without [body]  

### ISSUE THREAD

**Issue by hhaaii** [reactiveui/refit#1351]
(Apology this topic may be discussed before)
I am reading the https://github.com/reactiveui/refit#body-content document and understand what Body attribute when used for the 4 possibilities.

My question is for the example in above document, what is the behavior if I take the Body attribute from the "user" parameter. Will the user still be sent as the HTTP POST request body to the remote service endpoint?

_[Post("/users/new")]
Task CreateUser(User user);_

Thanks.
Hai

**Comment by glennawatson** [comment]
Yes. With Task CreateUser(User user) and no other body-eligible parameter, Refit treats the single complex-type parameter as the request body and serializes it (JSON by default). The [Body] attribute is only required to disambiguate when there are multiple candidates or to choose a serialization method (e.g. UrlEncoded). For a single object parameter on a POST it's implicit.

This is a usage question, so closing here. For follow-ups please use Discussions: https://github.com/reactiveui/refit/discussions


<blockquote><img src="https://opengraph.githubassets.com/7da521d756b94831ff31d8d5fc75ecbe2c2d69a69928da79595847192e775278/reactiveui/refit" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/reactiveui/refit/discussions">reactiveui refit · Discussions</a></strong></div><div>Explore the GitHub Discussions forum for reactiveui refit. Discuss code, ask questions & collaborate with the developer community.</div></blockquote>
